src/System.Private.CoreLib/shared/System/Globalization/CultureInfo.Windows.cs

```diff
@@ -41,15 +37,53 @@ private static CultureInfo GetUserDefaultCultureCacheOverride()
                 return (CultureInfo)callbacks.GetUserDefaultCulture();
             }
 #endif
+#if FEATURE_APPX
+            if (ApplicationModel.IsUap)
```

Comment:
I'm fine with moving it back to CultureInfo.cs. It seemed reasonable to keep it in the Windows specific code since that's what CoreRT was already doing, but I didn't look at the performance aspect.